application/models.py

```diff
@@ -34,6 +34,9 @@ class HighSchoolApplication(models.Model):
     program = models.ForeignKey(Program, on_delete=models.CASCADE)
     is_draft = models.BooleanField(default=True)
     submitted_date = models.DateTimeField()
+    application_status = models.IntegerField(
+        default="2"
```

Comment:
Since the field is of type "Integer" maybe we don't need the "2" and it should be a number 2.